REUSE.toml

```diff
@@ -200,6 +200,11 @@ path = "ui/src/assets/spdx-logo-color.svg"
 SPDX-FileCopyrightText = "2011 The Linux Foundation"
 SPDX-License-Identifier = "CC-BY-3.0"
 
+[[annotations]]
+path = "ui/src/assets/favicon.svg"
```

Comment:
Please move this to before "ui/src/assets/spdx-logo-color.svg" to maintain alphabetical sorting by path.